Example/Example.Android/Example.Android.csproj

```diff
@@ -67,6 +67,36 @@
     <Reference Include="AeroGear.Mobile.Core.Platform.Android">
       <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
     </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
+    <Reference Include="AeroGear.Mobile.Core.Platform.Android">
+      <HintPath>..\..\Core\Core.Platform.Android\bin\Debug\AeroGear.Mobile.Core.Platform.Android.dll</HintPath>
+    </Reference>
```

Comment:
Hmm, not sure what's going on here but it looks like these are duplicated refs

Reply:
Nice catch, not sure how it happened but yep I'll remove them